tool/tool-tool/su/src/java/org/sakaiproject/tool/su/SuTool.java

```diff
@@ -373,6 +377,18 @@ public void setUserinfo(User userinfo)
 		this.userinfo = userinfo;
 	}
 
+	public String getUserCreatedTime()
+	{
+		if (userinfo == null)
+		{
+			return "";
+		}
+
+		DateFormat dsf = DateFormat.getDateTimeInstance(DateFormat.LONG, DateFormat.LONG, msgs.getLocale());
+		dsf.setTimeZone(userTimeService.getLocalTimeZone());
+		return dsf.format(userinfo.getCreatedDate());
```

Comment:
IS it possible for userinfo to be not-null but getCreatedDate to be null (e.g., provided user)?

Reply:
I'm not sure. On our 11.3 production system a provided user will show the current time for creation time. I can amend the PR to copy that behaviour.

Reply:
I have no preference how created time should show for a provided user. I guess I'd prefer it not be shown at all but no strong pref. How does it work after this patch?

Reply:
I just changed it to output the current time if getCreatedDate() returns null. Assuming it does return null for provided users, it will show the same as in 11.

Date formatting matches what is currently used in the Users tool for the same data.